audio.py

```diff
@@ -0,0 +1,36 @@
+from pydub import AudioSegment
+
+
+def load_mp3(mp3_file_name):
+    """
+    This tries to load the audio from a named mp3 file.
+    It checks the filename has mp3 extension.
+    """
+    if mp3_file_name.lower().endswith('.mp3'):
```

Comment:
Reverting the condition would do early handling of the error and leave all proper work in the first level of indentation, removing the need for an else clause.